src/adapter/etl-adapter-parquet/src/Flow/ETL/Adapter/Parquet/ParquetLoader.php

```diff
@@ -14,31 +10,28 @@
 use Flow\ETL\Pipeline\Closure;
 use Flow\ETL\Row\Schema;
 use Flow\ETL\Rows;
+use Flow\Parquet\Options;
+use Flow\Parquet\Writer;
 
 /**
  * @implements Loader<array{
- *   path: Path,
- *   rows_in_group: int
+ *   path: Path
  * }>
  */
 final class ParquetLoader implements Closure, Loader, Loader\FileLoader
 {
-    private array $buffer;
-
     private readonly SchemaConverter $converter;
 
     private ?Schema $inferredSchema = null;
 
-    private ?ParquetDataWriter $writer = null;
+    private ?Writer $writer = null;
 
     public function __construct(
         private readonly Path $path,
-        private readonly int $rowsInGroup = 10000,
+        private readonly Options $options,
```

Comment:
This change requires to be applied also on examples & benchmark files.

Reply:
It was applied on the benchmarks and examples, the reason why benchmarks are not working is related to the fact that NativeEntryFactory while reading the parquet file is identifying uuid strings and creating UuidEntry which returns Uuid object when Rows::toArray is called since it's just iterating over entries and calling `$entry->value()` on it.